cmd/bbolt/page_command.go

```diff
@@ -251,6 +292,14 @@ func (cmd *PageCommand) PrintPage(w io.Writer, r io.ReaderAt, pageID int, pageSi
 func (cmd *PageCommand) Usage() string {
 	return strings.TrimLeft(`
 usage: bolt page PATH pageid [pageid...]
+   or: bolt page --all PATH
+
+Additional options include:
+
+	--all
+		prints all pages (only skips pages that were considered successful overflow pages) 
+	--value-format=`+FORMAT_MODES+` (default: auto)
+		just prints a length of the value, to reduce length and protect privacy.
```

Comment:
It seems that `--value-format` is only used to format the leaf page?

Reply:
Yes, as on other pages we have usually 'keys'. 

It makes sense - especially in etcd case to keep keys as hex.

Expanded the comment (it was clearly wrong).